src/Tizen.NUI/src/public/Window.cs

```diff
@@ -1311,6 +1311,58 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
             Interop.Window.delete_Window(swigCPtr);
         }
 
+        private static Dictionary<int, internalHookCallbackType> frameCallbackList = new Dictionary<int, internalHookCallbackType>();
+
+        private static readonly object locker = new object();
+
+        private static int key = 0;
+
+        private static FrameCallbackType internalHookFrameCallback = onInternalHookFrameCallback;
+
+        private struct internalHookCallbackType
+        {
+            public FrameCallbackType userCallback;
+            public int frameId;
+        }
+
+        private static void onInternalHookFrameCallback(int id)
```

Comment:
I think it is better to start with an upper-case letter.

Reply:
I agree with @rabbitfor, and I hope other private methods would also be started with upper-case.

Reply:
Thank you for your reviews. 
I changed only methods as pascal case. 
how do you think the following article about c# coding convention?
https://www.c-sharpcorner.com/UploadFile/8a67c0/C-Sharp-coding-standards-and-naming-conventions/
shall we make this as our internal rule? or is there any rule already proposed internally? ^^;
I think we need to make some our ground rule for NUI coding convention soon.

Reply:
I also hope to set up NUI coding convention asap. Anyway thanks for modification.